addons/sale/migrations/9.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,197 @@
+---Fields in module 'sale'---
+sale         / account.analytic.line    / so_line (many2one)            : NEW relation: sale.order.line
```

Comment:
That is, it replaces the to_invoice marker on the analytic line. So we should indeed create sale orders for existing lines with this marker set, as the migration of hr_timesheet_invoice which is no longer present). I guess doing it in the sale module is as good as any location.